src/irisnet/noncore/icelocaltransport.cpp

```diff
@@ -556,7 +556,7 @@ private slots:
         //                      .arg(toAddress.toString())
         //                      .arg(toPort));
 
-        sock->writeDatagram(packet, toAddress, toPort);
+        sock->writeDatagram(packet, toAddress, quintptr(toPort));
```

Comment:
please check no port use cast to quintptr. It's always quint16. There are a few more later, I won't comment on them.